pkgs/top-level/perl-packages.nix

```diff
@@ -10400,6 +10400,9 @@ let
     };
     propagatedBuildInputs = [ ArchiveZip DBFile FileWhich IOString ImageSize JSONXS LWP ParseRecDescent TextUnidecode URI XMLLibXML XMLLibXSLT shortenPerlShebang ];
```

Comment:
`perl 5.32.0` was recently merged to master, which no longer has  `Pod::Parser` in core. 

```suggestion
    propagatedBuildInputs = [ ArchiveZip DBFile FileWhich IOString ImageSize JSONXS LWP ParseRecDescent PodParser TextUnidecode URI XMLLibXML XMLLibXSLT shortenPerlShebang ];
```

Reply:
it is done in #97098

Reply:
@volth Ok, would it be best to wait until #97098 hits master in that case?

Reply:
or merge this to staging

Reply:
@stigtsp @volth do you need me to do something in this PR?